### DISCUSSION TITLE
Feature request: Dependency parser

### DISCUSSION THREAD

**Discussion by wholenews** [winkjs/wink-nlp#32] [New Features & Ideas]
🙏 Thanks for 😉 !!

**Comment by rachnachakraborty** [comment]
Hi @wholenews 

Glad to know that you find winkNLP useful. We will be adding this into our plan shortly; however it will take a while before we can commence development on this.

In the meantime, you may consider using [`.learnCustomEntities()`](https://winkjs.org/wink-nlp/learn-custom-entities.html) API to extract NOUN/VERB phrases. 

Cheers,
Rachna
